simple_certmanager/templates/admin/simple_certmanager/change_list.html

```diff
@@ -0,0 +1,8 @@
+{% extends "admin/change_list.html" %}
+
+{% load admin_list i18n %}
+
+{% block object-tools-items %}
+<li><a href="{% url 'generate_certificate' %}">Generate Certificate</a></li>
```

Comment:
Can we not just use the standard admin views to add a certificate? Why do we need a custom view?